### DISCUSSION TITLE
Automated CD

### DISCUSSION THREAD

**Discussion by Rufus125** [EventFahrplan/EventFahrplan#647] [Ideas]
Hey!
Finally, I achieved to automatically deploy to Google Play Store if a new version of the Eventfahrplan app is pushed to our main branch.
I thought this might be also interesting for the upstream project.

https://github.com/linuxtage/EventFahrplan/blob/glt-2022/.github/workflows/deploy.yaml

Documentation can be found here: https://github.com/r0adkll/upload-google-play?tab=readme-ov-file#configure-service-account

**Comment by johnjohndoe** [comment]
@Rufus125 Thank you for sharing! That is definitely a cool automation. I will have to see if it is useful to the current workflow because I regularly deploy different product flavors and not every tagged version is uploaded to Google Play for each of them.

I will move your post into the discussion section to keep track of it there.
